src/platform_impl/windows/dark_mode.rs

```diff
@@ -108,11 +94,12 @@ fn set_dark_mode_for_window(hwnd: HWND, is_dark_mode: bool) -> bool {
     type SetWindowCompositionAttribute =
         unsafe extern "system" fn(HWND, *mut WINDOWCOMPOSITIONATTRIBDATA) -> BOOL;
 
-    #[allow(non_snake_case)]
+    #[allow(clippy::upper_case_acronyms)]
     type WINDOWCOMPOSITIONATTRIB = u32;
     const WCA_USEDARKMODECOLORS: WINDOWCOMPOSITIONATTRIB = 26;
 
     #[allow(non_snake_case)]
+    #[allow(clippy::upper_case_acronyms)]
```

Comment:
```suggestion
    #[allow(non_snake_case, clippy::upper_case_acronyms)]
```
I think this should also work, but I'm not 100% sure which is best.

Reply:
I prefer two lines. Easier to read.

Reply:
Sure.